jwql/edb/engineering_database.py

```diff
@@ -61,6 +62,7 @@
 from bokeh.models import BoxAnnotation, ColumnDataSource, DatetimeTickFormatter, HoverTool, Range1d
 from bokeh.plotting import figure, output_file, show, save
 import numpy as np
+import tempfile
```

Comment:
Yikes, I hate seeing imports that weren't previously there!

Reply:
ME TOO! But tempfile is used in this module but was never imported 😓